cmsplugin_articles_ai/models/articles.py

```diff
@@ -106,6 +114,10 @@ class Article(models.Model):
         verbose_name=_("URL slug"),
         db_index=True,
     )
+    language = LanguageField(
```

Comment:
I think we settled that the view can be language-neutral and just render every language.